customScripts/expansiontravel.gd

```diff
@@ -177,6 +185,8 @@ func getzonetraveltext(zone,progress):
 			array.append("There's writing on the wall here.  In some dialect of elvish, it reads: "+randomfromarray(["Abandon hope any who enter.","Turn around cousin on pain of death, we serve you no longer.","We tolerate no intruders.","No Trespassing","Do Not Trespass","Scenic Cavern Ahead","Don't Feed the Ooze","Disclaimer: This enclave of dark elves is not responsible for dismemberment or death of any party trespassing herein."]))
 		else:
 			array.append("There's writing on the wall here.  "+randomfromarray(["It's some dialect of elvish, but you cannot make it out","It's illegible to you, but an arrow pointing onward with an 'X' carved through it and skulls convey the meaning well enough."]))
+	elif zone.code == 'undercityruins':
+		array.append_array(["Your party turns a corner and comes face to face with a wall.  You turn around to backtrack and dozens of sightless eyes glisten in the darkness from alcoves you couldn't see on your way into this dead end.\n\nYou prepare for the end as one massive trogolodyte moves to the center of the tunnel ahead of the others, but before it can act a Kobold wearing some sort of jersey drops from the ceiling and bravely slaps the monstrosity on its nose as it falls before scurrying through the trog's legs and behind it.\n\nThe Kobold races down the hall screaming it's warcry, [color=yellow]Got your nose![/color].  The enraged trogolodytes forget all about your party and give chase, leaving you alone in relative safety.\n\nYou're relatively certain that the Kobold's jersey read: 'Capitualize'."])
```

Comment:
This entry does not need append_array() as it is a single string

Reply:
comment incorporated on new commit